includes/rest-api.php

```diff
@@ -755,13 +754,10 @@ public function pmpro_rest_api_recent_memberships( $request ) {
 
 			$results = $wpdb->get_results( $wpdb->prepare( $sql, $limit ) );
```

Comment:
Since it seems like you're expecting a single row to be returned (based on line 817, for instance), what about being more specific in the DB access code and use $wpdb->get_row( $wpdb->prepare( $sql, $limit ) ); ?

Reply:
It could be more than one but initial support is based around the first one for Zapier purposes. The API could still be used to get multiple results by providing the 'limit' parameter.